python/glow/gwas/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .lin_reg import *
-__all__ = ['linear_regression']
\ No newline at end of file
+from .log_reg import *
+
+__all__ = lin_reg.__all__ + log_reg.__all__
```

Comment:
Huh, I'm surprised this works. I didn't think that `import * from foo` imports `foo` itself.